.github/workflows/label.yml

```
@@ -0,0 +1,18 @@
+---
+name: "Label PRs from globs"
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  execute:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: jpmcb/prow-github-actions@v1.1.3
```

https://github.com/jpmcb/prow-github-actions/releases/tag/v2.0.0

```suggestion
      - uses: jpmcb/prow-github-actions@v2.0.0
```

---

Yes, I wanted the #72 to pick this and the others up ;-)

I can update this one manually if you think that should be that way

---

If you can, please update to latest, I will star your dependabot  :-)